_data/landscape/meshes.yml

```diff
@@ -311,6 +348,8 @@
   encryption: "Yes"
   multi-tenant-score: 1
   netdev-persona-score: 5
+  spiffe: "?"
```

Comment:
No, but the community is exploring migrating to SPIFFE https://github.com/openservicemesh/osm/issues/1681